src/utils/relativeTime.ts

```diff
@@ -0,0 +1,35 @@
+
```

Comment:
maybe we should delete all this and just use momentjs?

Reply:
I did look at momentjs and from what I have read it has a reputation for having a large footprint. I was unsure if it was warranted for such a small use case. I will switch to using momentjs because it seems like something devs will be more familiar with.

Reply:
yeh i have heard the same. i'm not sure if they have a more minimal include now or if there's something better, but if not then i agree familiarity and less code is likely better even so